pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+### Sidebar Checklist
+
+- [ ] Request reviewers
+- [ ] Assign yourself and other contributors
+- [ ] Link to project
+
+### Issues Resolved
+Resolves #
+
+### Problem Addressed
+
+### Solution Implemented
+
+### Other Notes
```

Comment:
I like this template! A little different than ones I've used in the past, but I think it will fit well into the project.

Reply:
This looks interesting. I think it will really help our process.